bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/connector/UplinkWebInterface.java

```diff
@@ -90,12 +90,13 @@
      *
      * @author afriese - initial contribution
      */
+    @NonNullByDefault
     private class WebRequestExecutor implements Runnable {
 
         /**
          * queue which holds the commands to execute
          */
-        private final Queue<NibeUplinkCommand> commandQueue;
+        private final Queue<@Nullable NibeUplinkCommand> commandQueue;
```

Comment:
I do not want to allow null commands, but Collections to not work well when using these annotations. For example if I do not annotate this with @Nullable the "poll" command is assumed to always return nonnull values. But this is not true, it will in fact return null when the queue is empty.